utils/defaults.stanza

```diff
@@ -63,25 +63,24 @@ public pcb-board default-board (num-layers:Int, outline:Shape) :
 
 ; Creates a design using the default board. 
 public defn make-default-board (module:Instantiable, num-layers:Int, outline:Shape) :
-  
+  set-rules(ocdb/rules/bay-area-circuits-conservative-rules)
```

Comment:
Shouldn't we remove this from here all together if we need to call it in the design file anyway? Or is it just for rectangular boards and some other shapes that we need it in the design file?

Reply:
Other designs than tutorial and mcu work thanks to this call to set-rules in here?

Reply:
I don't know actually. I think its hazardous because it silently overwrites rule in subsequent calls. So maybe we should remove it.